roles/contiv_network/templates/aci_gw.j2

```diff
@@ -19,7 +19,7 @@ start)
     -e "APIC_EPG_BRIDGE_DOMAIN={{ apic_epg_bridge_domain }}" \
     -e "APIC_CONTRACTS_UNRESTRICTED_MODE={{ apic_contracts_unrestricted_mode }}" \
     --name=contiv-aci-gw \
-    contiv/aci-gw
+    {{ aci_gw_version }}
```

Comment:
Is there any way in ansible so that i can have default version as : contiv/aci-gw and if user passes different version then it should overwrite and take this new version ?